Jenkinsfile.cloud

```diff
@@ -70,16 +70,21 @@ node(env.NODE) {
         }
     }
 
-    def commit, dirpath, qcow, vmdk
+    def commit, version, dirpath, qcow, vmdk
     stage("Finalizing") {
         def image = sh(returnStdout: true, script: "ls /var/lib/imagefactory/storage/*.body").trim()
         // just introspect after the fact to avoid race conditions
         commit = sh(returnStdout: true,
                     script: """LIBGUESTFS_BACKEND=direct virt-cat -a ${image} -m /dev/coreos/root:/ \
                                     /ostree/repo/refs/remotes/rhcos/${ref}""").trim()
-        currentBuild.description = 'commit ' + commit
+        version = sh(returnStdout: true,
+                     script: "ostree show --repo=${repo} --print-metadata-key=version ${commit}").trim()
```

Comment:
Ah wow, that's some heroics!  I was going to reply that it was racy, but I see what you're doing now.